_run/kube/Makefile

```diff
@@ -9,6 +9,7 @@ GATEWAY_ENDPOINT ?= http://$(GATEWAY_HOST)
 provider-run:
 	AKASH_DEPLOYMENT_INGRESS_STATIC_HOSTS="false" \
 		$(AKASHCTL) $(KEY_OPTS) provider run \
+			--chain-id "$(CHAIN_NAME)" \
```

Comment:
I think this is in `$(KEY_OPTS)`.

Reply:
Its not which is why I had to add this. We used to pass it in via `akashctl config`, now needs to be added manually. Would be nice if the SDK picked that up off of the genesis file...

Reply:
maybe it's been changed since then but it looks like it's included [now](https://github.com/ovrclk/akash/pull/781/files/2a91040524a235ddbd992b34ede7c712f8806db4#diff-a4a9755559e6a7113337c16c483d84d8R18)

agreed that not having the config command makes this really cumbersome.

Reply:
I have trouble thinking about the Makefile import cycle sometimes 🤦 thank you for finding that